docs/source/conf.py

```diff
@@ -0,0 +1,97 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(os.path.join("..", "..", "src")))
+
+version = "main"
+
+# -- Project information -----------------------------------------------------
+
+project = "MaRDA Metadata Extractors"
+copyright = "2021 - 2023, MaRDA Metadata Extractors Team"
```

Comment:
```suggestion
copyright = "2022 - 2023, MaRDA Metadata Extractors Team"
```

Let's cut ourselves some slack on how long this has taken